packages/modals/src/styled/StyledFooterItem.ts

```diff
@@ -7,15 +7,24 @@
 
 import styled from 'styled-components';
 import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
-import ModalStyles from '@zendeskgarden/css-modals';
 
 const COMPONENT_ID = 'modals.footer_item';
 
 export const StyledFooterItem = styled.span.attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: ModalStyles['c-dialog__footer__item']
+  'data-garden-version': PACKAGE_VERSION
 })`
+  display: flex;
+  /* stylelint-disable property-no-unknown */
+  margin-${props => (props.theme.rtl ? 'right' : 'left')}: ${props => props.theme.space.base * 5}px;
+  /* stylelint-enable property-no-unknown */
+  min-width: 0;
+
+  &:first-child {
+    margin-right: ${props => props.theme.rtl && 0};
+    margin-left: 0;
```

Comment:
Structure this just as you did the one-line `margin-...` property above. In other words, `margin-${props.theme.rtl ? ... }: 0;` This will always be the preferred pattern when you encounter directionality margining/padding/etc.